src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Details from './components/Details/Details';
+import Header from './components/Header/Header';
+import Home from './components/Home/Home';
+import Search from './components/Search/Search';
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/details" element={<Details />} />
```

Comment:
- [ ] The category item detail page has its unique route within the SPA like an ID